### ISSUE TITLE
ValueError: dict contains fields not in fieldnames: 'DeepImmuno WT IC50 Score', 'DeepImmuno MT IC50 Score'

### ISSUE THREAD

**Issue by YingYa** [griffithlab/pVACtools#1075]
### Installation Type

Docker

### pVACtools Version / Docker Image

4.1.0 - running via Singularity (v3.8.1)

### Operating System

CentOS7

### Describe the bug

**Running pVACtools using Singularity on a CentOS Linux 7 HPC at work. (Docker not permitted)**

**The key information from the stdout:**
Executing MHC Class I predictions
......
Parsing prediction file for Allele HLA-B*15:01 and Epitope Length 9 - Entries 1-104

**The error information from stderr:**
Traceback (most recent call last):
  File "/usr/local/bin/pvacseq", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/main.py", line 123, in main
    args[0].func.main(args[1])
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/run.py", line 142, in main
    pipeline.execute()
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/pipeline.py", line 452, in execute
    split_parsed_output_files = self.parse_outputs(chunks)
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/pipeline.py", line 413, in parse_outputs
    parser.execute()
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/output_parser.py", line 755, in execute
    tsv_writer.writerow(row)
  File "/usr/local/lib/python3.7/csv.py", line 155, in writerow
    return self.writer.writerow(self._dict_to_list(rowdict))
  File "/usr/local/lib/python3.7/csv.py", line 151, in _dict_to_list
    + ", ".join([repr(x) for x in wrong_fields]))
ValueError: dict contains fields not in fieldnames: 'DeepImmuno WT IC50 Score', 'DeepImmuno MT IC50 Score'

### How to reproduce this bug

```Shell
singularity exec -e -B /path/to/indir:/indir -B `pwd` /path/to/pvactools-4.1.0.sif pvacseq run /indir/test2.vcf.gz test2 HLA-A*02:01,HLA-B*15:01,HLA-B*57:01,HLA-C*03:03,HLA-C*06:02,DPA1*01:03,DPA1*02:02,DPB1*02:01,DPB1*19:01,DQA1*01:03,DQA1*02:01,DQB1*03:03,DQB1*06:03,DRB1*07:01,DRB1*13:01 all . --iedb-install-directory /opt/iedb -t 4 --minimum-fold-change 1 --tdna-vaf 0.1 --trna-vaf 0.1 -a sample_name --normal-sample-name normal2 --tumor-purity 0.25 -p /indir/test2.phased.vep.vcf.gz
```

### Log output

Traceback (most recent call last):
  File "/usr/local/bin/pvacseq", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/main.py", line 123, in main
    args[0].func.main(args[1])
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/run.py", line 142, in main
    pipeline.execute()
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/pipeline.py", line 452, in execute
    split_parsed_output_files = self.parse_outputs(chunks)
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/pipeline.py", line 413, in parse_outputs
    parser.execute()
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/output_parser.py", line 755, in execute
    tsv_writer.writerow(row)
  File "/usr/local/lib/python3.7/csv.py", line 155, in writerow
    return self.writer.writerow(self._dict_to_list(rowdict))
  File "/usr/local/lib/python3.7/csv.py", line 151, in _dict_to_list
    + ", ".join([repr(x) for x in wrong_fields]))
ValueError: dict contains fields not in fieldnames: 'DeepImmuno WT IC50 Score', 'DeepImmuno MT IC50 Score'

**Comment by susannasiebert** [comment]
@YingYa thank you for this bug report. Can you please attach the input VCF file for this run so I can try to reproduce this error on my end?

**Comment by YingYa** [comment]
Yes. May I have your e-mail that I could share the files by google drive?

**Comment by susannasiebert** [comment]
Would our Google group help email alias work? It’s help@pvactools.org

**Comment by susannasiebert** [comment]
Unfortunately, I'm unable to reproduce this issue on my end. Would you be able to share your output directory for this run with me if you still have it? Additionally, the full stderr/stdout log would be helpful in case there are any additional errors/warnings in there.

**Comment by susannasiebert** [comment]
I was actually able to reproduce this error on the data for #1074. Did you, by any chance, restart this particular run? Did you encounter the following error earlier: `FileNotFoundError: [Errno 2] No such file or directory: '/data/issue_1074/MHC_Class_I/tmp/deepimmuno-cnn-result.txt'`? I'm trying to figure out if this error is related to a restart after the FileNotFoundError.

**Comment by YingYa** [comment]
Yes, I encounter the following error in other rerun samples:  _FileNotFoundError: [Errno 2] No such file or directory: '/path/to/MHC_Class_I/tmp/deepimmuno-cnn-result.txt'_

**Comment by susannasiebert** [comment]
This issue has been fixed in version 4.1.1. You will need to start your run from scratch because the `FileNotFoundError` results in incomplete tmp files that will lead to the `ValueError` from the original bug report. Please give that version a try and feel free to reopen this issue if the error persists.
